__tests__/atomic-swap/sign.test.js

```diff
@@ -0,0 +1,154 @@
+import hathorLib from '@hathor/wallet-lib';
+import { ProposalInput, ProposalOutput, PartialTx } from '@hathor/wallet-lib/lib/models/partial_tx';
+import TestUtils from '../test-utils';
+
+const walletId = 'stub_atomic_swap_tx_proposal_sign';
+
+describe('tx-proposal sign api', () => {
```

Comment:
This test file is exactly the same as the `sign-and-push` file. We should reuse all the methods only changing the API, so we don't have this duplicated code

Reply:
I'll wait the decision on changing the `sign-and-push` api to just `push` before changing this.